spec/lib/vcr/library_hooks/faraday_spec.rb

```diff
@@ -43,8 +43,8 @@
 
   it 'prints a warning if the faraday connection stack contains a middleware after the HTTP adapter' do
     conn = Faraday.new(:url => 'http://sushi.com') do |builder|
-      builder.use Faraday::Adapter::NetHttp
-      builder.use Faraday::Response::Logger
+      builder.adapter Faraday::Adapter::NetHttp
+      builder.adapter Faraday::Response::Logger
```

Comment:
I believe this is a Middleware, whereas the line before has an Adapter.

Reply:
Ah, thanks, let me try again 😆